pkgs/tools/misc/colord/default.nix

```diff
@@ -26,7 +26,7 @@
 , docbook_xml_dtd_412
 , gtk-doc
 , libxslt
-, enableDaemon ? !stdenv.hostPlatform.isMusl || stdenv.hostPlatform.isStatic
+, enableDaemon ? true
```

Comment:
The option only got added for musl so we could remove it again https://github.com/NixOS/nixpkgs/commit/46339db687cb7daf92af20db1c14152ade29c4a1

Reply:
We could, but it's not hurting anything to leave it in, and it would be useful for e.g. closure size reduction.

Reply:
I am just a bit worried about it bit rotting